semantic-source/src/Source/Source.hs

```diff
@@ -50,7 +51,7 @@ import           Source.Span (Span(Span), Pos(..))
 -- 'ByteString' under the hood. Construct these with 'fromUTF8'; obviously,
 -- passing 'fromUTF8' non-UTF8 bytes will cause crashes.
 newtype Source = Source { bytes :: B.ByteString }
-  deriving (Eq, Semigroup, Monoid, IsString, Show, Generic)
+  deriving (Eq, Semigroup, Monoid, IsString, Show, Generic, NFData)
```

Comment:
I dig this, but I’ll admit I’m pretty curious what prompted it specifically 😊

Reply:
I was benchmarking things that returned `Source` values. I elided them in the final patch, since they’re internal functions rather than external behaviors and thus aren’t useful to benchmark over the long term, but I figured it’s the right thing for `Source` to have `NFData` anyway.

Reply:
Ahhhhh I see! Yeah, great call.